kubernetes/jobs-initcontainers-cronjobs/intro.md

```diff
@@ -13,7 +13,7 @@ There're two types of jobs:
 [Init Containers](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/) are regular containers within a pod that run before the app container and they also satisfy the following statements:
 
 * They can run setup scripts not present in an app container - e.g prepopulate some data, waiting until a specific service is up and running and etc.
-* A pod can have one or more init containers apart of app containers
-* They always run to completation
+* A pod can have one or more init containers apart from app containers
+* Init containers always run to completation
 * Each one must complete successfully before the next one is started
-* Application container won't run if any init container won't finish successfully
+* The application container won't run if any of the configured init containers will not finish the execution successfully
```

Comment:
can you add an example here of the CronJob Pod consisting of multiple containers - one being SQLProxy which the  other container may rely on to complete. However CronJob will never `complete` because SQLProxy will always be un a running state despite the cronjob script completing it's task. It's a Very common example.